### ISSUE TITLE
Please, switch to Linux Kernel 4.19.x branch

### ISSUE THREAD

**Issue by alfieoriginal** [maxnet/berryboot#551]
in order to improve compatibility. Thanks

**Comment by XxAcielxX** [comment]
Yes, I'll agree with this too. I'm using LibreELEC and I want to play HEVC files which won't work with older Kernel.

Even the Raspbian has the newer kernel released few days ago.

**Comment by maxnet** [comment]
>Yes, I'll agree with this too. I'm using LibreELEC and I want to play HEVC files which won't work with 
>older Kernel.

Don't think you need a new kernel for that.
Just newer firmware files should do.

Can grab those manually from: https://github.com/raspberrypi/firmware/tree/master/boot
start*.elf fixup*.dat

**Comment by maxnet** [comment]
BTW do note that the ones shipped with Berryboot are less than a month old (11 december)
I don't really see a need to update daily. New is not necessarily better.

**Comment by XxAcielxX** [comment]
Alright, I did update those 8 files and then tested this file mentioned in this post:
https://forum.kodi.tv/showthread.php?tid=298461&pid=2806215#pid2806215

Which others have said runs smooth, but it doesn't for me. What could be the reason?

**Comment by maxnet** [comment]
>Which others have said runs smooth

And those others have the same Pi model as you?
And with or without overclocking?

**Comment by XxAcielxX** [comment]
I have RPi3 and some others as well, so it's a yes. About OC, I can't say.

**Comment by XxAcielxX** [comment]
https://forum.kodi.tv/showthread.php?tid=298461&pid=2806497#pid2806497

Milhouse says it's LE under Berryboot.

I asked Milhouse if there is anything to get full LE support under Berryboot. Awaiting his reply now.

**Comment by macmpi** [comment]
So far LE team has not been very keen on Berryboot integration.
I've worked-out simple install & integration scripts you will find [there](https://github.com/macmpi/berryboot-scripts)
(note I will soon update them, but they work as-is)
